modules/ROOT/pages/organization.adoc

```diff
@@ -66,8 +71,8 @@ Organization-level client IDs and client secrets are supported only for backward
 === View the Client ID and Secret for Organizations and Environments
 
 . Sign into your Anypoint Platform account as a user with the Organization Administrators role.
-. Select the *Access Management* option from the main page, or select it from the left side bar by clicking the _menu_ button in the top left corner.
-. Click *Organization*, or *Environments* in the left menu.
+. Select the *Access Management* option from the main page, or click the menu button in the top left corner to access it from the left side bar.
```

Comment:
"it" isn't clear here. "the option" would make it clear. Or, we can just give them one option.  They'll discover the menu method on their own?